coderd/database/migrations/000159_workspace_agent_stats_template_id_created_at_user_id_idx.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX workspace_agent_stats_template_id_created_at_user_id_idx ON workspace_agent_stats USING btree (template_id, created_at DESC, user_id) WHERE connection_count > 0;
```

Comment:
Should we add a `COMMENT ON INDEX ... IS 'Support index for insights ...';` for posterity?

Reply:
Good call!